### DISCUSSION TITLE
我想问一下，Simplicity StudioV4是不是已经办法下载了？

### DISCUSSION THREAD

**Discussion by FlyyingPiggy2020** [MarkDing/IoT-Developer-Boot-Camp#35] [Q&A]
![image](https://user-images.githubusercontent.com/62139687/127443954-99476fcb-a1e9-48e8-9975-2e25532105bd.png)
因为我看了这里的教程是基于V4制作的，所以想去官方网站上下个v4版本的Studio，但是貌似那个V4的按钮没办法互动。

**Comment by FlyyingPiggy2020** [comment]
这个是官网的下载链接，但是我下不了v4,不知道是我网络的问题还是什么其他原因。[链接](https://www.silabs.com/developers/simplicity-studio)

**Comment by FlyyingPiggy2020** [comment]
![image](https://user-images.githubusercontent.com/62139687/127446253-49b15a29-cc0c-475f-b426-46c92e4008dc.png)
在点亮LED章节里面，v4版本有选择EmberNet Soc的选项，我想问一下v5版本中，貌似没有这个选项了。我应该怎么配置这个，还是说不用配置了。

**Comment by MarkDing** [comment]
这个教程是基于v4的，现在应该都是v5了，你还是基于V5来开发吧。有问题可以在silabs的论坛询问，不好意思造成你的困惑

发自我的iPhone

在 2021年7月29日，下午3:02，BeeeeeeLazy ***@***.***> 写道：

﻿

[image]<https://user-images.githubusercontent.com/62139687/127446253-49b15a29-cc0c-475f-b426-46c92e4008dc.png>
在点亮LED章节里面，v4版本有选择EmberNet Soc的选项，我想问一下v5版本中，貌似没有这个选项了。我应该怎么配置这个，还是说不用配置了。

**Reply by FlyyingPiggy2020** [reply]
好的谢谢您的回复，这个教程还是很有意义的。因为刚拿到这块板子，之前也没接触过zigbee，要是没这些教程，程序都还下不进去呢😂。
